diff --git a/node_modules/conf/dist/source/index.d.ts b/node_modules/conf/dist/source/index.d.ts
index 70d6dd0..966f6b8 100644
--- a/node_modules/conf/dist/source/index.d.ts
+++ b/node_modules/conf/dist/source/index.d.ts
@@ -2,7 +2,6 @@
 import { EventEmitter } from 'events';
 import { OnDidChangeCallback, Options, Unsubscribe, Schema, OnDidAnyChangeCallback } from './types';
 declare class Conf<T extends Record<string, any> = Record<string, unknown>> implements Iterable<[keyof T, T[keyof T]]> {
-    #private;
     readonly path: string;
     readonly events: EventEmitter;
     constructor(partialOptions?: Readonly<Partial<Options<T>>>);
